src/amber/cli/templates/app/config/routes.cr

```diff
@@ -15,6 +16,7 @@ Amber::Server.configure do |app|
   # All static content will run these transformations
   pipeline :static do
     plug Amber::Pipe::PoweredByAmber.new
+    #plug Amber::Pipe::ClientIp.new(["X-Forwarded-For"])
```

Comment:
This probably doesn't need to be here, I don't think the static pipeline would need this under normal circumstances.

Reply:
Yes and no. If you are running behind a load balancer and want the exact same behavior that you would have if you were receiving requests directly, you would enable this.

Also I would think it makes sense for the default behavior to be uniform, which would call for enabling (or leaving disabled) ClientIp pipe in both places. If individual people know specific reasons why they want to enable in one and not another, that should be their skilled and circumstantial decision.

Reply:
I can't see myself ever using it, but I also don't see any harm to leaving it here. My thought is: if it's serving a static file, why is there any reason to be looking at the client ip address. But I suppose that isn't all encompassing.

Reply:
@robacarp - true, the client IP is mostly irrelevant there. But it comes handy for logging, and also for the element of least surprise. Otherwise a developer could make 1 request in their browser (e.g. to reload the page) and see two different IPs being reported in Amber.

Reply:
> Otherwise a developer could make 1 request in their browser (e.g. to reload the page) and see two different IPs being reported in Amber, one for dynamic content and one for static content.

And that would mess with logging. Gotcha!